server/context.go

```diff
@@ -165,12 +166,12 @@ func (ctx *Context) initEngine(attrsStr, path string) (engine.Engine, error) {
 	return engine.NewRocksDB(attrs, path, ctx.CacheSize), nil
 }
 
-// parseAttributes parses a colon-separated list of strings,
+// parseAttributes parses a list of strings separated by sep,
 // filtering empty strings (i.e. ",," will yield no attributes.
 // Returns the list of strings as Attributes.
-func parseAttributes(attrsStr string) proto.Attributes {
+func parseAttributes(attrsStr string, sep string) proto.Attributes {
```

Comment:
Well commas separate the stores and colon separates multiple attributes within a store.

For example:

-store=hdd:7200rpm=/mnt/hda1,ssd:gpu=/mnt/ssd01

Would mean one store with attributes {hdd, 7200rpm} at /mnt/hda1 and another store with attributes {ssd, gpu} at /mnt/ssd01.